src/python/chevahbs

```diff
@@ -131,56 +120,14 @@ chevahbs_install() {
-                --extract $destination/python27.dll.embedded \
-                $destination/python27.dll
-
-            echo "Patching manifests to use our redistributable version"
-            execute sed -e 's|version="9.0.21022.8"|version="9.00.30729.6161"|' -e 's|publicKeyToken="1fc8b3b9a1e18e3b"||' < $destination/python.exe.embedded > $destination/python.exe.manifest
-            execute sed -e 's|version="9.0.21022.8"|version="9.00.30729.6161"|' -e 's|publicKeyToken="1fc8b3b9a1e18e3b"||' < $destination/pythonw.exe.embedded > $destination/pythonw.exe.manifest
-            execute sed -e 's|version="9.0.21022.8"|version="9.00.30729.6161"|' -e 's|publicKeyToken="1fc8b3b9a1e18e3b"||' < $destination/python27.dll.embedded > $destination/python27.dll.manifest
-            execute rm -f --verbose $destination/*.embedded
-
-            echo "Installing pip + binary modules"
-            execute $destination/python $get_pip $PIP_ARGS
-            execute $destination/python -m pip \
-                install $PIP_ARGS $PIP_INSTALLABLES_WIN
-            # Remove pip.
-            execute $destination/python -m pip uninstall -y pip
-
-            echo "Patching pyWin32 manifests to use our redistributable version"
-            $manifest_wiper --verbose \
-                --extract $destination/Lib/site-packages/win32/pythonservice.exe.embedded \
-                $destination/Lib/site-packages/win32/pythonservice.exe
-            $manifest_wiper --verbose \
-                --extract $destination/Lib/site-packages/win32/perfmondata.dll.embedded \
-                $destination/Lib/site-packages/win32/perfmondata.dll
-            $manifest_wiper --verbose \
-                --extract $destination/Lib/site-packages/pywin32_system32/pythoncomloader27.dll.embedded \
-                $destination/Lib/site-packages/pywin32_system32/pythoncomloader27.dll
-
-            execute sed -e 's|version="9.0.21022.8"|version="9.00.30729.6161"|' -e 's|publicKeyToken="1fc8b3b9a1e18e3b"||' < $destination/Lib/site-packages/win32/pythonservice.exe.embedded > $destination/Lib/site-packages/win32/pythonservice.exe.manifest
-            execute sed -e 's|version="9.0.21022.8"|version="9.00.30729.6161"|' -e 's|publicKeyToken="1fc8b3b9a1e18e3b"||' < $destination/Lib/site-packages/win32/perfmondata.dll.embedded > $destination/Lib/site-packages/win32/perfmondata.dll.manifest
-            execute rm -f --verbose $desination/Lib/site-packages/win32/*.embedded
-            execute sed -e 's|version="9.0.21022.8"|version="9.00.30729.6161"|' -e 's|publicKeyToken="1fc8b3b9a1e18e3b"||' < $destination/Lib/site-packages/pywin32_system32/pythoncomloader27.dll.embedded > $destination/Lib/site-packages/pywin32_system32/pythoncomloader27.dll.manifest
-            execute rm -f --verbose $desination/Lib/site-packages/pywin32_system32/*.embedded
-
-            echo "Copy Python runtime to pyWin32 package"
-            execute cp $destination/*CRT.manifest $destination/Lib/site-packages/win32/
```

Comment:
this was moved inside chevah_build in the `command_build_python_extra_libraries` section as this is about fixing pywin32 lib, and is not about the core python version